spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/SimpleRouteLocator.java

```diff
@@ -83,15 +92,23 @@ public SimpleRouteLocator(String servletPath, ZuulProperties properties) {
 	}
 
 	@Override
-	public Route getMatchingRoute(final String path) {
-
-		return getSimpleMatchingRoute(path);
+	public Route getMatchingRoute(HttpServletRequest request) {
+		return getSimpleMatchingRoute(request);
+	}
 
+	@Override
+	public Route getMatchingRoute(final String path) {
+		// Nothing to do here, will do matching using RequestContext
```

Comment:
Either this should continue to work or throw an exception, doing nothing is a bad experience for a developer expecting this to work.